pages/platform/ai/notebook_tuto_10_create_chatbot/guide.en-gb.md

```diff
@@ -0,0 +1,118 @@
+---
+title: AI Notebooks - Tutorial - Create, Modify and train a Rasa chatbot
```

Comment:
_I would have shortened the title (for SEO):_

title: AI Notebooks - Tutorial - Create and train a Rasa chatbot